test/Profiler/coverage_inlined_counters_exec.swift

```diff
@@ -21,6 +21,9 @@
 // REQUIRES: profile_runtime
 // REQUIRES: executable_test
 
+// FIXME: Currently fails on non-Darwin (https://github.com/swiftlang/swift/issues/75240)
+// REQUIRES: OS=macosx
```

Comment:
Is this affected by the changes in this patch? It looks unrelated to me?

Reply:
This is because we're now running this test on non-Darwin platforms, previously we wouldn't have picked up the `profile_runtime` feature